File: control-assurance-sp/src/types/GIAAActionOwner.ts
import { IEntity } from "./Entity";

export interface IGIAAActionOwner extends IEntity {
    GIAARecommendationId?: number;
    UserId?: number;
}

export class GIAAActionOwner implements IGIAAActionOwner {
    public ID = 0;
    public Title = null;
    public GIAARecommendationId = null;
    public UserId = null;
}

File: ControlAssuranceAPI/Models/GIAAActionOwner.cs
using System;
using System.Collections.Generic;

namespace CAT.Models;

public partial class GIAAActionOwner
{
    public int ID { get; set; }

    public string? Title { get; set; }

    public int? GIAARecommendationId { get; set; }

    public int? UserId { get; set; }

    public DateTime? DateAssigned { get; set; }

    public virtual GIAARecommendation? GIAARecommendation { get; set; }

    public virtual User? User { get; set; }
}
